app/assets/javascripts/controllers/ems_physical_infra_dashboard/physical_infra_aggregate_status_card_controller.js

```diff
@@ -0,0 +1,63 @@
+/* global miqHttpInject */
+
+angular.module( 'patternfly.card' ).controller('physicalInfraStatusCardController', ['providerId', 'API', 'miqService', function(providerId, API, miqService) {
+  var vm = this;
+  var attributes = ["physical_servers", "physical_racks"];
+
+  var attrIconHsh = {
+    "physical_servers": "pficon pficon-cluster",
+    "physical_racks": "pficon pficon-enterprise",
+  };
+
+  var attrUrl = {
+    "physical_servers": "physical_servers",
+    "physical_racks": "physical_racks",
+  };
+
+  var getIcon = function getIcon(providerType) {
+    var type = providerType.split("::");
+    return type[2].toLowerCase();
+  };
+
+  var getUrl = function(entity) {
+    return providerId + "?display=" + attrUrl[entity];
+  };
+
+  var getAttributesData = function() {
+    vm.status = {
+      "iconImage": "/assets/svg/vendor-" + getIcon(vm.provider.type) + ".svg",
```

Comment:
This doesn't work outsite development environment.

Every single image url **must** be constructed server-side, using the `image_path` helper.

Reply:
Cool @himdel I will try to make it work for both :)